<?xml version="1.0" encoding="UTF-8"?>

<launch>

    <include file="$(find rotw7_pkg)/launch/run_turtle_tf_broadcaster_kalman.launch"/>
    <include file="$(find rotw7_pkg)/launch/run_turtle_tf_listener_45degree.launch">
        <arg name="model_to_be_followed_name" value="turtle1" />
        <arg name="follower_model_name" value="turtle2" />
    </include>
    
</launch>